packages/orbit-components/src/Textarea/README.md

```diff
@@ -34,6 +34,7 @@ Table below contains all types of the props available in Textarea component.
 | resize      | [`enum`](#enum)            | `"vertical"` | The resize option for Textarea.                                                                                                                                |
 | rows        | `number`                   |              | Specifies the height of the text area (in lines).                                                                                                              |
 | readOnly    | `boolean`                  |              | Adds readOnly to html textarea element.                                                                                                                        |
+| required    | `boolean`                  |              | Optional prop, adds required to html textarea                                                                                                                  |
```

Comment:
This is not consistent with how we usually write, at some point lack of code tags can become confusing. In this case "required" is also a regular word, so in order to help it become perceived as the name of the HTML attribute, we can put it in code tags.

We should generally write nicer, for example "HTML" is an acronym so it should be uppercase, but never mind that now.
```suggestion
| required    | `boolean`                  |              | Optional prop, adds `required` to the `<textarea>` HTML element.                                                                                               |
```